network/address.go

```diff
@@ -122,6 +122,10 @@ func (a Address) Host() string {
 	if e != nil {
 		return ""
 	}
+	// IPv6 global address has to be in brackets.
```

Comment:
Minor nitpick: This comment is confusing. Should be rather "unspecified address" instead of "global adress"